File: functions/notes.py
#Cairo Taylor, Functions Notes

#Function is action stored by keyword

def add(num1, num2):
    print(num1+num2)

#add(12, 20)
#add(14, 5)
#add(int(input("Tell me a number:\n")), 45)

def user(word):
    answer = input(f"Tell me a {word}:\n")
    return answer

name = user("name").capitalize()
verbing = user("present tense verb").lower()
place = user("place").capitalize()
print(f"{name} was {verbing} to {place}.")

File: functions/notes.c
//cairo Taylor, Functions Notes
#include <stdio.h>

void add(int num1, int num2){
    printf("%d\n", num1+num2);
}

const char* input(char type[50], int length){
    char answer[50];
    printf("Give me a %s:\n", type);
    getStr(answer, sizeof(answer)-1);
    return answer;
}

int main(void){
   //printf("Hello World\n");
   //add(12, 9);
   input("name", 50);
   input("verb", 50);
   input("place", 50);
   printf("%s was %s until they somehow reached %s", input("name", 50), input("verb", 50), input("place", 50));
   return 0;
}